catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/SolrMetacardClientImpl.java

```diff
@@ -260,7 +260,8 @@ public SourceResponse query(QueryRequest request) throws UnsupportedQueryExcepti
         addDocsToResults(docs, results);
 
         if (userSpellcheckIsOn && solrSpellcheckHasResults(solrResponse)) {
-          query.set("q", findQueryToResend(query, solrResponse));
+          Collation collation = findQueryToResend(query, solrResponse);
```

Comment:
✏️ Possibly worth changing the method name to maybe something like findBestCollation or something that matches the return value more appropriately.